### PR TITLE
SUBMARINE-748. Update Contributing guide

### PR DESCRIPTION
**Pull request by kevin85421**
### What is this PR for?
Update Contributing guide

### What type of PR is it?
[Documentation]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-748

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kevin85421** [comment]
Use this link to preview contributing.md.
https://github.com/kevin85421/hadoop-submarine/blob/SUBMARINE-748/website/docs/community/contributing.md

**Comment by yaooqinn** [comment]
It looks more like a tool guide for developers to make changes in the source code but not a contributing guide.

In the contributing guide, I suggest that we might encourage people to contribute not only in the way of committing to the source code but also testing, discussing/answering questions on the mailing list, doc improvement, sharing use cases, and so on.

**Comment by kevin85421** [comment]
@yaooqinn Thank you for your recommendation! I will update it soon.

**Comment by kevin85421** [comment]
I have already updated my pull request. Use this link to preview contributing.md.
https://github.com/kevin85421/hadoop-submarine/blob/SUBMARINE-748/website/docs/community/contributing.md

File: website/docs/community/contributing.md [hunk 1]
```diff
@@ -15,252 +15,132 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 ```sh
-git push ${your_remote_name} myrevert
+git push origin SUBMARINE-${jira_number} 
 ```
 
-Create a PR based on this branch.
-
-#### Cherry pick a commit to a release branch
+### Step8: Check Travis-ci status of your personal fork
+* Visit `https://travis-ci.org/github/${USERNAME}/submarine`
```
Comment (pingsutw) [inline website/docs/community/contributing.md:107]:
```suggestion
* Visit `https://travis-ci.com/github/${USERNAME}/submarine`
```

**Review by pingsutw — COMMENTED** [review]
Rest LGTM, Thanks

**Pull request closed without merging**